x-pack/plugins/cloud_security_posture/server/plugin.ts

```diff
@@ -51,6 +52,7 @@ import {
 export interface CspAppContext {
   logger: Logger;
   service: CspAppService;
+  security: SecurityPluginStart;
```

Comment:
sidenote: 

we should [prioritize](https://github.com/orgs/elastic/projects/705/views/8?filterQuery=route+handler) adding a route handler context and not build on our own context.